__tests__/app.test.js

```diff
@@ -35,14 +35,14 @@ describe('Incorrect file path', () => {
 describe('/api/articles/articles_id', () => {
     test('200: Get responds with an article object', () => {
         return request(app).get('/api/articles/1').expect(200).then(({body}) => {
-            expect(body.article).toMatchObject({
+            expect(body.article).toEqual(expect.objectContaining({
                 title: "Living in the shadow of a great man",
                 topic: "mitch",
                 author: "butter_bridge",
                 body: "I find this existence challenging",
                 created_at: '2020-07-09T20:11:00.000Z',
                 votes: 100,
-            })
+            }))
```

Comment:
You shouldn't need to amend your original test in this case as that does not follow the rules of TDD. 

Yes, objectContaining is a good way to go as you can have the original test for the earlier ticket in place, which will allow for there to be extra keys on the object we are not currently testing.
Then add an extra test to assert the comment_count behaviour. This is seen frequently when adding complexity as features. We do not want to change our previous tests as they will give us the confidence to make the required changes whilst still passing the previous tests.